Gruntfile.js

```diff
@@ -141,8 +146,9 @@ module.exports = function (grunt) {
 	grunt.registerTask('coverageReport', ['exec:coverageReport']);
 	grunt.registerTask('eslint-nofix', ['eslint']);
 	grunt.registerTask('test', ['eslint', 'exec:coverage']);
-	grunt.registerTask('test-unit', ['eslint', 'exec:coverageUnit']);
-	grunt.registerTask('test-functional', ['eslint', 'exec:testFunctional']);
+	grunt.registerTask('testUnit', ['eslint', 'exec:testUnit']);
+	grunt.registerTask('testFunctional', ['eslint', 'exec:testFunctional']);
```

Comment:
Because it is a small change for development purposes and the discussion started thanks to this PR. @Isabello do really you want me to remove these changes and open a new issue/PR ?